src/styles/components/protectedLayout.less

```
@@ -101,122 +113,133 @@
-                        color: inherit;
                         flex-shrink: 0;
                     }
 
-                    .menuIcon svg {
-                        width: 2rem;
-                        height: 2rem;
+                    svg {
+                        width: 28px;
+                        height: 28px;
                         color: inherit;
                         fill: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)